dubbo-rpc/dubbo-rpc-rest/src/main/java/org/apache/dubbo/rpc/protocol/rest/RpcContextFilter.java

```diff
@@ -85,4 +92,14 @@ public void filter(ClientRequestContext requestContext) throws IOException {
             requestContext.getHeaders().add(DUBBO_ATTACHMENT_HEADER, attachments);
         }
     }
+
+    /**
+     * If a string value illegal for rest protocol(',' and '=' is illegal for rest protocol).
+     *
+     * @param v string value
+     * @return true for illegal
+     */
+    private boolean illegalForRest(String v) {
+        return v.contains(",") || v.contains("=");
```

Comment:
Or judge null place here ?

Reply:
Prefer judge null in 'illegalForRest'.
The meaning of this method is that if a value is not null, it should not contains ',' or '='.
I will fix this soon.
:)